README.md

```diff
@@ -19,6 +19,7 @@ Here we have:
 - [shell](shell): Examples for how to run Radix shell, which can enable spinning up temporary interactive
    nodes. The Radix Shell code itself is in [cli-tools](cli-tools).
 - [docker](docker): An option for running a network of nodes locally. You can also use the "Run Single Validator" IntelliJ option.
+- [testnet-node](testnet-node): The easiest way to setup a development environment for integrators (check the [README](testnet-node/README.md))
```

Comment:
I'd suggest to make it really easy to find, we should have an ## Integrators section even above `## SubDirectories` - and say - if you're looking to run a node for developing an integration, see `[details on running a testnet node for development](./testnet-node)` and the `[Babylon documentation for exchanges and integrators](https://docs.google.com/document/d/1cjc7_alyzIb2QQIGGn1PEpJyjrMRZYHq3VwkOXRP8J0/edit#)`